tip-wlan/values.yaml

```diff
@@ -73,9 +73,15 @@ global:
     password: tip-read
 
   monitoring:
+    jmxExporterAgentUrl: https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.14.0/jmx_prometheus_javaagent-0.14.0.jar 
```

Comment:
@brennerm are we pulling this during runtime? Maybe worth creating a Docker image for this one and use it instead of the alpine mentioned down the line in this values file?

Reply:
How are we going to access the jar file from the application container in this case? :thinking:

Reply:
@leonidlm ping

Reply:
@brennerm not 100% sure about the current implementation, but we can mount a shared ephemeral volume between two containers under one pod and just copy the jar from one filesystem to another. Saw this done multiple times in other cases.

Another option is to add this JAR to our CloudSDK Docker images, no?

Reply:
So we would have a Docker image that contains only the JAR file? Where's the benefit of this approach over downloading the agent from a URL on startup?

Yup, that would be another option.

Reply:
@leonidlm I'm currently waiting for Bitnami to add the Java agent JAR to their Docker image to be able to use it here. According to https://github.com/bitnami/bitnami-docker-jmx-exporter/issues/1 they will be adding it soon.